src/libs/actions/Session/index.js

```diff
@@ -397,22 +383,6 @@ function changePasswordAndSignIn(authToken, password) {
         });
 }
 
-/**
- * Call set or change password based on if we have an auth token
- * @param {Number} accountID
- * @param {String} validateCode
- * @param {String} password
- * @param {String} authToken
- */
-
-function setOrChangePassword(accountID, validateCode, password, authToken) {
-    if (authToken) {
-        changePasswordAndSignIn(authToken, password);
-        return;
-    }
-    setPassword(password, validateCode, accountID);
-}
```

Comment:
Just removing this because I don't think it's clear to have a single method responsible for setting a password or changing one and this can be left up to the view to decide.